File: floriday-staff/src/app/models/product.ts
import { ProductCategories } from './enums';
import { BaseModel } from './base.model';

export class Product extends BaseModel {
    Name: string;
    Price: string;
    ImageUrl: string;
    ProductCategories: ProductCategories;
    Page: number;
}

export class Products {
    Products: Product[];
}

File: Florid/Florid.Schema/Entities/Product.cs
using System;
using Florid.Enum;

namespace Florid.Entity
{
    public class Product : BaseEntity
    {
        public string Name { get; set; }
        public string Price  { get; set; }
        public string ImageUrl { get; set; }
        public ProductCategories productCategories { get; set; }
        public int Page { get; set; }
    }
}
